db/migrate/20171006134319_empty_refunds.rb

```diff
@@ -0,0 +1,13 @@
+class EmptyRefunds < ActiveRecord::Migration
```

Comment:
Do we need this migration?

Reply:
It was there to zap the data because I had a migration that changed the db level integrity checks (added null checking) - without this, the migration would fail.

Im not a big fan of changing migrations once they are in there, but maybe this one is an exception ?

Reply:
Yes, I think it can be confusing for devs if we keep it. Also, since this does not affect the db schema then it's safe to remove.

Reply:
have now removed it